woocommerce/woocommerce
# added states to and  Locale for Portugal

### All Submissions:

* [x ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?


### Changes proposed in this Pull Request:
This PR proposes changing:


Portugal was on the list of countries without the state field. Option removed
I came across this problem today when I installed woocommerce. 

Portugal was on the list of countries without the state field. Option removed.
Portugal was on the list of countries without the state field,  has been added states to portugal.

### How to test the changes in this Pull Request:

1. install/activate woocommerce
2. To choose Portugal as Store Base
3. states/ districts of the portugal will automatically appear

### Other information:

* [ x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ x] Have you written new tests for your changes, as applicable?
* [ x] Have you successfully ran tests with your changes locally?

### Changelog entry

has been added states to portugal.
